src/app/_types/metadataTypes.ts

```diff
@@ -4,15 +4,15 @@ type OpenGraphMetadata = {
 }
 
 type TwitterMetadata = {
-  title?: string
-  card?: string
+  card?: 'summary' | 'summary_large_image' | 'player' | 'app'
```

Comment:
Do we need all of these? `player` and `app` may not apply to us. Not sure.

Reply:
Yea `app` probably not, but `player` is used for media and video? Or you would just leave `summary` and `summary_large_image`?